server/test/services/yeelight/api/yeelight.controller.test.js

```diff
@@ -10,9 +10,16 @@ const res = {
 };
 
 describe('YeelightController GET /service/yeelight/discover', () => {
-  it('should call discover', async () => {
+  it('should call discover and send all the devices', async () => {
     const yeelightController = YeelightController(yeelightService);
     await yeelightController['get /api/v1/service/yeelight/discover'].controller({}, res);
     assert.called(yeelightService.discover);
   });
+
+  it('should call discover and send nothing (too much call api)', async () => {
+    const yeelightController = YeelightController(yeelightService);
+    yeelightService.discoveryInProgress = true;
+    await yeelightController['get /api/v1/service/yeelight/discover'].controller({}, res);
+    assert.called(yeelightService.discover);
+  });
```

Comment:
This test is the same as the previous one, except that the variable "discoveryInProgress " is set to true.
I don't think he should call discover, should he?
Maybe you can add that res.json is called with or without arguments...

Reply:
Yes anyway due to the yeelight controller but not interesting ^^

Reply:
I'll remove it